nixos/modules/services/web-servers/h2o/default.nix

```
@@ -171,19 +171,24 @@ let
                 # ACME is the most likely use case.
                 #
                 # See: https://github.com/mozilla/ssl-config-generator/issues/323
-                tlsRecAttrs = lib.optionalAttrs hasTLSRecommendations (
-                  let
-                    recs = mozTLSRecs.${tlsRecommendations};
-                  in
-                  {
-                    min-version = builtins.head recs.tls_versions;
-                    cipher-preference = "server";
-                    "cipher-suite-tls1.3" = recs.ciphersuites;
-                  }
-                  // lib.optionalAttrs (recs.ciphers.openssl != [ ]) {
-                    cipher-suite = lib.concatStringsSep ":" recs.ciphers.openssl;
+                # See: https://letsencrypt.org/2024/12/05/ending-ocsp/
+                tlsRecAttrs =
+                  lib.optionalAttrs (builtins.elem names.cert certNames.all) {
```

I don't understand when this would evaluate to false? How could we be dealing with a cert that isn't in the list of all certs? Is this possible because `certNames.all` is actually just the certs that have ACME enabled, and it's possible that `names.cert` is for a certificate we obtained some other way?

---

Per our discussion on matrix. How would you feel about moving the link you added above to a comment here:

```suggestion
                  # # If we're using ACME to get the cert, disable OCSP: https://letsencrypt.org/2024/12/05/ending-ocsp/
                  lib.optionalAttrs (builtins.elem names.cert certNames.all) {
```